src/sim_util.py

```diff
@@ -183,9 +183,12 @@ def update_schedule_with_latency(schedule, latency):
     for edge in schedule.edges:
         node = edge[0]
         func = schedule.nodes.data()[node]["function"]
+        scaling = 1
         if func == "stall":
             func = node.split("_")[3]
-        schedule.edges[edge]["weight"] = latency[func]
+        if func == "Buf" or func == "MainMem":
+            scaling = 16
```

Comment:
why 16 here? does this have anything to do with hardcoded value on 411?

Reply:
related. But good catch. This I could actually update back to `data['size']`.
On line 411, the node data doesn't yet have the size attribute, so I had to hardcode the 16 for all values. Need to at some point go back and fix that and have the data sizes loaded from the instrumentation, but for now they're all set as 16 bits.